e2e/change_license_test.go

```diff
@@ -0,0 +1,74 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/replicatedhq/kots/e2e/cluster"
+	"github.com/replicatedhq/kots/e2e/kubectl"
+	"github.com/replicatedhq/kots/e2e/testim/inventory"
+	"github.com/replicatedhq/kots/e2e/util"
+	"github.com/replicatedhq/kots/e2e/workspace"
+)
+
+var _ = Describe("Change License", func() {
```

Comment:
I would have to play with the code to figure out the feasibility but it seems like we would want to have all the tests in one file inside various contexts and have reach be it's own test(). Below you have a decent amount of abstracted setup with the before,after,context but only one test.

Reply:
I experimented with table driven tests but i gave up. i agree as most code is duplicate boilerplate in each test file.